### PR TITLE
feat(yapily): add bank error states

### PR DESCRIPTION
**Pull request by TheLeoB**
## Description (optional)
Additional error codes/copy to bank status flyout. 
## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankStatus/template.error.general.tsx [hunk 1]
```diff
@@ -71,6 +80,21 @@ const BankLinkError: React.FC<Props> = ({
...
+          ) : bankStatus === 'BANK_TRANSFER_ACCOUNT_REJECTED' ? (
+            <FormattedMessage
+              id='copy.bank_linked_error_title_connectionrejected'
+              defaultMessage='Connection Rejected'
+            />
+          ) : bankStatus === 'BANK_TRANSFER_ACCOUNT_FAILED' ? (
+            <FormattedMessage
+              id='copy.bank_linked_error_title_failedconnection'
+              defaultMessage='Failed Connection Request'
+            />
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankStatus/template.error.general.tsx:97]: maybe we can move all of bank statuses to be enum and then we can just reuse it all around 

```
export enum BANK_STATUSES {
  BANK_TRANSFER_ACCOUNT_EXPIRED = 'BANK_TRANSFER_ACCOUNT_EXPIRED',
  BANK_TRANSFER_ACCOUNT_REJECTED = 'BANK_TRANSFER_ACCOUNT_REJECTED',
 // etc...
}
```

and then just use 

```
bankStatus === BANK_STATUSES.BANK_TRANSFER_ACCOUNT_REJECTED
```

**Review by milan-bc — APPROVED** [review]
Looks good to me I added just one suggestion it could be added as part of this PR or added later not a bloker ;)

**Review by pedroapfilho — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankStatus/template.error.general.tsx (hunk 1 above)
Comment (TheLeoB) [reply packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankStatus/template.error.general.tsx:99]: Done, thanks!

File: packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/types.ts [hunk 2]
```diff
@@ -56,12 +56,16 @@ interface OBAttributesType {
   institutions: OBInstitution[]
 }
 
-export type BankStatusType =
-  | 'ACTIVE'
-  | 'BANK_TRANSFER_ACCOUNT_INFO_NOT_FOUND'
-  | 'BANK_TRANSFER_ACCOUNT_ALREADY_LINKED'
-  | 'BANK_TRANSFER_ACCOUNT_NAME_MISMATCH'
-  | 'DEFAULT_ERROR'
+export enum BankStatusType {
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/types.ts:59]: 👏🏻

**Review by blockdylanb — APPROVED** [review]

**Pull request merged**